admin-dev/themes/new-theme/js/components/translatable-input.js

```diff
@@ -40,10 +46,9 @@ class TranslatableInput {
         const form = localeItem.closest('form');
         const selectedLocale = localeItem.data('locale');
 
-        form.find('.js-locale-btn').text(selectedLocale);
-
-        form.find('input.js-locale-input').addClass('d-none');
-        form.find('input.js-locale-input.js-locale-' + selectedLocale).removeClass('d-none');
+        form.find(this.localeButtonSelector).text(selectedLocale);
+        form.find(this.localeInputSelector).addClass('d-none');
+        form.find(this.localeInputSelector+'.js-locale-' + selectedLocale).removeClass('d-none');
```

Comment:
Missing space before `.js-locale-`?

Reply:
no there are several classes on the input, `js-locale-input` is the common one `js-locale-fr` allows to target the input by language